_pages/test-pilot.md

```diff
@@ -11,6 +11,16 @@ redirect_from: "/experiments/test-pilot/"
 * TOC
 {:toc}
 
+{: .label .timebox}
+<span class="glyphicon glyphicon-time" aria-hidden="true"></span> 2 Hours
```

Comment:
Would you mind exchanging this with "2 hours" (lowercase h) to make it consistent with the other pages?